packages/desktop-client/src/components/common/Link.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { type ReactNode, type ComponentProps } from 'react';
+import { NavLink, useMatch, useNavigate } from 'react-router-dom';
+
+import { css } from 'glamor';
+
+import { type CSSProperties, styles } from '../../style';
+
+import Button from './Button';
+
+type ButtonLinkProps = ComponentProps<typeof Button> & {
+  to: string;
+  activeStyle?: CSSProperties;
+};
+
+type AnchorLinkProps = {
+  to: string;
+  style?: CSSProperties;
+  activeStyle?: CSSProperties;
+  children?: ReactNode;
+};
+
+const ButtonLink = ({
+  to,
+  style,
+  activeStyle,
+  onClick,
+  ...props
+}: ButtonLinkProps) => {
+  const navigate = useNavigate();
+  const match = useMatch({ path: to });
+
+  const handleClick = e => {
+    onClick?.(e);
+    navigate(to);
+  };
+
+  return (
+    <Button
+      style={{
+        ...style,
+        ...(match ? activeStyle : {}),
+      }}
+      activeStyle={activeStyle}
+      {...props}
+      onClick={handleClick}
+    />
+  );
+};
+
+const AnchorLink = ({ to, style, activeStyle, children }: AnchorLinkProps) => {
+  const match = useMatch({ path: to });
+
+  return (
+    <NavLink
+      to={to}
+      className={`${css([
+        styles.smallText,
+        style,
+        match ? activeStyle : null,
+      ])}`}
+    >
+      {children}
+    </NavLink>
+  );
+};
+
+type LinkProps = {
+  to: string;
+  linkType?: 'button' | 'anchor';
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`linkType` for `Link` seems unnecessarily verbose. What if we just changed it to `type`?

Reply:
the button link component has a prop named `type` as well.

Reply:
Good point. Maybe we can call it `variant`? Open to other suggestions though.